### ISSUE TITLE
Language change buttons not tabbable

### ISSUE THREAD

**Issue by Sendouc** [Sendouc/sendou.ink#916]
![Image](https://user-images.githubusercontent.com/38327916/188477508-27ce1860-e17e-4a94-83ec-ccdb0e2cdfd8.png)

This should be accessible via keyboard only + show focus ring.
